client/constants.js

```diff
@@ -1,3 +1,5 @@
+// TODO Organize this file by reducer type, ot break this apart into
+// multiple files
```

Comment:
Yeah this file has gotten crazy, and can definitely be broken apart. Not sure if it would be better to put this in a folder called "constants" or maybe follow the "ducks" redux design pattern.